src/runtime/pkg/govmm/qemu/qemu.go

```diff
@@ -2471,14 +2471,26 @@ const (
 	Unix QMPSocketType = "unix"
 )
 
-// QMPSocket represents a qemu QMP socket configuration.
+const (
+	// Socket using a human-friendly text-based protocol.
+	Hmp string = "hmp"
+
+	// Socket using a richer json-based protocol.
+	Qmp string = "qmp"
+
+	// Same as Qmp with pretty json formatting.
+	QmpPretty string = "qmp-pretty"
+)
+
+// QMPSocket represents a qemu QMP or HMP socket configuration.
 // nolint: govet
 type QMPSocket struct {
 	// Type is the socket type (e.g. "unix").
 	Type QMPSocketType
 
-	// Human Monitor Interface (HMP) (true for HMP, false for QMP, default false)
-	IsHmp bool
+	// Protocol is the protocol to be used on the socket (either Hmp, Qmp or
+	// QmpPretty).
+	Protocol string
```

Comment:
You could consider using the widely used pattern in govmm that's also used here for `QMPSocketType` - declare something like `type QMPSocketProtocol string` and make the protocol consts and this new `Protocol` member be of type `QMPSocketProtocol`.  This would also make the explanation in the comment unnecessary.

Reply:
I'll give a try.

Reply:
Done